test/unit test/Storage_service_test.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:sinix_android/services/Storage_service.dart';
+
+void main() {
+  test('Local Storage test', () async {
+    final LocalStorage localStorage = LocalStorage();
+
+    await localStorage.init();
+
+    // firstOpen will always be true in test
+    expect(localStorage.firstOpen, true);
```

Comment:
This test is failing for me. Can you try this out once more and update.

Reply:
Yes, there was a mistake by me.
actually, I didn't know that we have to initialize shared preferences with mock data when we are testing it.